frontend/src/components/sprint/UserTable.js

```diff
@@ -31,7 +31,7 @@ const UserTable = ({list, username}) =>
         </thead>
         <tbody>
         {list.map(item =>
-            <tr key={item.key} className={"table-row"}>
+            <tr key={item.key} className={"table-row" + (item.is_flagged ? ' flagged' : '')}>
```

Comment:
Could be written using template literals which looks much cleaner to me than concat. But serves the same purpose -
```
<tr key={item.key} className={`table-row ${item.is_flagged ? 'flagged' : ''}`}>
```

Reply:
@shimulch, TBH nesting braces in one line looks less readable to me in this particular case. I'll leave it as is, because we might revisit it anyway after the UX review.